UnitedIncome/components
# Deployment Changes

### Description
> Please provide a brief description of your changes below.

Removes the AWS deployment job so it's now hosted on GitHub pages.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Once this pushes to dev we will see it build there. I may need to make a couple of minor changes.
* This does not replace the entire CircleCI job. It probably doesn't really need to, but it may be better to not have two CI tools.
